libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -787,8 +819,11 @@ mod test {
     #[test]
     fn test_high_rotation_is_rejected() {
         let (side_a_image, side_b_image, options) = load_ballot_card_fixture(
-            "2021-06-06-lincoln-test-ballot",
-            ("high-rotation-front.png", "high-rotation-back.png"),
+            "nh-test-ballot",
+            (
+                "template-rotated-3deg-front.jpeg",
```

Comment:
I created these images manually by rotating them beyond the allowed rotation limit (2º).